modules/configure-acs-policy-notifications.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-email.adoc
+:_module-type: PROCEDURE
+[id="configure-acs-policy-notifications"]
+= Configuring policy notifications for {product-title}
+
+You can use the following steps to configure policy notifications:
+
+.Procedure
+
+. Navigate to *Platform Configuration* *>* *System Policies*.
+. Select the policies for which you want to send alerts.
+. Select *Enable Notifications* or *Actions* *>* *Enable Notification* (this depends on the {product-title} platform version you are using).
+. In the *Enable Notifications* dialog, select the email notifier (same as the *Integration Name*). The *No notifiers configured!* message is displayed if you have not configured any other integrations.
+. Select *Enable*.
+
+[IMPORTANT]
+====
+
+* The {product-title-short} platform sends notifications on an opt-in basis. Assign a notifier to the policy to receive notifications.
+* For a given alert, notifications are sent one time. If you have assigned a notifier to a policy, you receive a notification only if a violation generates a new alert. The {product-title-short} platform creates a new alert when:
```

Comment:
IBMSG suggests "To avoid possible translation problems, use complete sentences and not phrases to introduce lists". 

Could you make the following sentence a complete sentence to introduce the list? Using the colon is fine: 

The {product-title-short} platform creates a new alert when

for example: 

The {product-title-short} platform creates a new alert during the following scenarios: 

You don't have ot use that one, just a suggestion.

Reply:
@gaurav-nelson  I made the changes in the file (**Files changed** tab) however, the preview shows the previous content. Can you see why the current preview does not reflect the change? The other changes show up correctly. I will take a look at this again on Tuesday, but want to ask you just in case you had time to look at the file...thanks!
https://deploy-preview-40742--osdocs.netlify.app/openshift-acs/latest/integration/integrate-with-email